controllers/node.go

```diff
@@ -592,8 +592,15 @@ func createNode(w http.ResponseWriter, r *http.Request) {
 				logic.ReturnErrorResponse(w, r, logic.FormatError(err, "internal"))
 				return
 			}
-			logic.UpdateHost(&data.Host, host) // update the in memory struct values
-
+			logic.UpdateHostFromClient(&data.Host, host) // update the in memory struct values
+			logger.Log(0, fmt.Sprintf("HOST ----------> %+v\n", data.Host))
+			err = logic.UpsertHost(host)
```

Comment:
this is going to overwrite the encoded password and cause issues going forward.

Reply:
no it won't

Reply:
he is maintaining the current host in db